src/jit/gentree.cpp

```diff
@@ -6111,6 +6331,14 @@ GenTreePtr          Compiler::gtCloneExpr(GenTree * tree,
             }
             break;
 
+        case GT_COPY:
+        case GT_RELOAD:
+            {
+                copy = new(this, oper) GenTreeCopyOrReload(oper, tree->TypeGet(), tree->gtGetOp1());
+                copy->AsCopyOrReload()->CopyOtherRegs(tree->AsCopyOrReload());
```

Comment:
I don't think you need this here, because you call CopyOtherRegs from CopyReg().